packages/account/src/Components/poi/idv-document-submit/utils.js

```diff
@@ -34,6 +34,12 @@ export const isSequentialNumber = document_number => {
     return pattern_results.includes(true);
 };
 
+export const isIDVWhitelistDocumentNumber = (country, document_type, document_number) =>
+    (idv_test_document_whitelist.has(country) &&
+        idv_test_document_whitelist.get(country)[document_type] === document_number &&
+        getPlatformFromUrl().is_test_link) ||
+    getPlatformFromUrl().is_staging;
```

Comment:
I think I also see a bug in your code, I believe you wrapped the closing bracket `)` in the wrong place. You put it around `getPlatformFromUrl().is_test_link) <-` and not around ` getPlatformFromUrl().is_staging` aswell.

Reply:
Thanks @hubert-deriv , I guess I miss it while refactoring, and my IDE made some auto adjustments :)